scripts/serve_api.py

```diff
@@ -0,0 +1,71 @@
+"""Serve SourceCred API, for consumption by Observable notebooks.
+
+Usage:
+
+    python3 ./scripts/serve_api.py [PORT]
+
+from the SourceCred repository root. Omit the port to try a default, or
+specify port 0 to pick a free port automatically.
+
+This server binds to `localhost`, which is a loopback address only
+accessible from your machine, not the local network.
+
+Corresponding Observable code:
+
+    sourcecred = {
+      const server = "http://localhost:7007";
+      const blob = await fetch(server).then((r) => r.blob());
+      const esModule = await require(URL.createObjectURL(blob));
+      return esModule.default;
+    }
```

Comment:
Can you add a note why are we using the blob URL?
I'm guessing it's to force we're not caching the file?
Similar to something like 
```js
require(`http://localhost:7070/api.js?t=${Date.now()}`)
```

Reply:
The primary reason is to prevent caching. A secondary reason is that it
hints at what to do if you’re serving from a CDN that doesn’t serve the
proper MIME types, like GitHub’s raw file view. (Namely, just wrap this
in `new Blob([b], {type: "application/javascript"})`. You can’t do that
if it’s a direct `require("http://...")`. I thought that this was worth
including because @decentralion _was_ trying to require directly from
GitHub—it’s a natural thing to do—and was running into errors.)

Reply:
Good tip, I hadn't tried executing js files from GitHub in this way, only json. So I hadn't run into the mime type issues.

Reply:
I will note that if one doesn’t care about these issues, one can write:

```
sourcecred = (await require("http:localhost:9009")).default
```

This works as long as the MIME type served is correct, and is a bit
shorter. So if @decentralion prefers I can use that as the recommended
incantation.

Reply:
Just bumped into a service that promises the solve the mime issue.
https://raw.githack.com/